packages/js-moi-logic/__tests__/logic.test.ts

```diff
@@ -17,8 +17,8 @@ const PROVIDER = new JsonRpcProvider(HOST);
 let wallet: Wallet;
 
 beforeAll(async () => {
-    wallet = new Wallet(PROVIDER);
-    await wallet.fromMnemonic(MNEMONIC, DEVIATION_PATH);
+    wallet = await Wallet.fromMnemonic(MNEMONIC, DEVIATION_PATH);
```

Comment:
what is DEVIATION_PATH.

Reply:
a36b11b60f5391536a65926f9450046806d84589